src/aiotk/tcp.py

```diff
@@ -99,16 +117,21 @@ def port(self):
         """
         return self._port
 
+    # NOTE: cannot declare type for return value here because the class
+    #       definition is not completed...
     async def __aenter__(self):
         self.start()
         await self.wait_started()
         return self
 
-    async def __aexit__(self, *args):
+    async def __aexit__(self, *args) -> Optional[bool]:
         self.close()
         await self.wait_closed()
+        # NOTE: `mypy` will not allow `return` without an expression even when
+        #       it accepts `None`...
+        return None
```

Comment:
**Note to self**: report this feedback to `mypy` devs to ensure they are aware of this limitation in `mypy`.